src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -378,6 +390,111 @@
+
+                        for (i = 0; i < scripts.length; ++i) {
+                            eval(scripts[i].firstChild.nodeValue);
+                        }
+
+                        el.style.display = -1 !== xhr.responseText.indexOf('sf-toolbarreset') ? 'block' : 'none';
+
+                        if (el.style.display == 'none') {
+                            return;
+                        }
+
+                        if (Sfjs.getPreference('toolbar/displayState') == 'none') {
+                            document.getElementById('sfToolbarMainContent-' + token).style.display = 'none';
+                            document.getElementById('sfToolbarClearer-' + token).style.display = 'none';
+                            document.getElementById('sfMiniToolbar-' + token).style.display = 'block';
+                        } else {
+                            document.getElementById('sfToolbarMainContent-' + token).style.display = 'block';
+                            document.getElementById('sfToolbarClearer-' + token).style.display = 'block';
+                            document.getElementById('sfMiniToolbar-' + token).style.display = 'none';
+                        }
+
+                        /* Handle toolbar-info position */
+                        var toolbarBlocks = [].slice.call(el.querySelectorAll('.sf-toolbar-block'));
+                        for (i = 0; i < toolbarBlocks.length; ++i) {
+                            toolbarBlocks[i].onmouseover = function () {
+                                var toolbarInfo = this.querySelectorAll('.sf-toolbar-info')[0];
+                                var pageWidth = document.body.clientWidth;
+                                var elementWidth = toolbarInfo.offsetWidth;
+                                var leftValue = (elementWidth + this.offsetLeft) - pageWidth;
+                                var rightValue = (elementWidth + (pageWidth - this.offsetLeft)) - pageWidth;
+
+                                /* Reset right and left value, useful on window resize */
+                                toolbarInfo.style.right = '';
+                                toolbarInfo.style.left = '';
+
+                                if (elementWidth > pageWidth) {
+                                    toolbarInfo.style.left = 0;
+                                }
+                                else if (leftValue > 0 && rightValue > 0) {
+                                    toolbarInfo.style.right = (rightValue * -1) + 'px';
+                                } else if (leftValue < 0) {
+                                    toolbarInfo.style.left = 0;
+                                } else {
+                                    toolbarInfo.style.right = '0px';
+                                }
+                            };
+                        }
+                        Sfjs.addEventListener(document.getElementById('sfToolbarHideButton-' + token), 'click', function (event) {
```

Comment:
use the `addEventListener` function directly now that you are inside the scope defining it

Reply:
We are in a callback, so I'd have to do something like `var that = this;` in the parent scope. Is there any value in doing that instead of using the existing `Sfjs`?

Reply:
No, I mean using the `addEventListener` which is available here (the one being assigned into `Sfjs.addEventListener` to expose it to the outside